### PR TITLE
Add SpaceID in Create Work Item for Kitemaker

### PR DESCRIPTION
**Pull request by l4u**

### CODE REVIEW

File: packages/nodes-base/nodes/Kitemaker/Kitemaker.node.ts [hunk 1]
```diff
@@ -119,10 +119,13 @@ export class Kitemaker implements INodeType {
 			},
 
 			async getStatuses(this: ILoadOptionsFunctions) {
+				const spaceId = this.getNodeParameter('spaceId', 0) as string;
+
 				const responseData = await kitemakerRequest.call(this, { query: getStatuses });
 				const { data: { organization: { spaces } } } = responseData;
+				const space = spaces.find((e: { [x: string]: string; }) => e.id == spaceId)
```
Comment (l4u) [inline packages/nodes-base/nodes/Kitemaker/Kitemaker.node.ts:126]: Is there a type definition for spaces?

**Comment by ivov** [comment]
Thank you for your contribution.
* You added the parameter, but you are not [including it in the request](https://github.com/kitemakerhq/docs/blob/cf395d19e90e0dedd31b2df2702943c2b52e5a60/kitemaker.graphql#L339).
* In `workItem:create`, when Space ID is empty, Status ID throws `TypeError: Cannot read property 'statuses' of undefined` in the console.

**Comment by l4u** [comment]
@ivov It seems that the statusID is associated with a spaceID. So, the item goes to the right space without the need to send the spaceID.

I've added a check for the spaceID input to provide a clearer error message.

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot for your contribution. Got merged and released with the next version.

**Comment by janober** [comment]
Got released with `n8n@0.156.0`

**Pull request merged**